man/waybar-mpris.5.scd

```diff
@@ -71,12 +71,27 @@ The *mpris* module displays currently playing media via libplayerctl.
 	something less than or equal to this value, so the title will always be ++
 	displayed.
 
-*dynamic-priority*: ++
+*dynamic-order*: ++
+	typeof: []string ++
+	default: ["title", "artist", "album", "position", "length"] ++
+	Order of the tags shown by Dynamic tag. The position and length tags ++
+	will always be combined in the format [{position}/{length}]. The order ++
+	of these tags in relation to other tags will be determined based on the ++
+	declaration of the first among the two tags. Absence in this list means ++
+	force exclusion.
+
+*dynamic-truncation-priority*: ++
```

Comment:
We need to keep `dynamic-priority` for backward compability

Reply:
I think it's fine to rename it in the manual, but check if the old name is set in code

Reply:
> I think it's fine to rename it in the manual, but check if the old name is set in code

done